app/_includes/plugins/rate-limiting/strategies.md

```diff
@@ -0,0 +1,47 @@
+
+The {{page.name}} plugin supports three rate limiting strategies: `local`, `cluster`, and `redis`. 
+{% if page.name == "Rate Limiting Advanced" %}
+This is controlled by the `config.strategy` parameter.
+{% elsif page.name == "Rate Limiting" %}
+This is controlled by the `config.policy` parameter.
+{% endif %}
+
+| Strategy    | Pros | Cons   |
+| --------- | ---- | ------ |
+| `local`   | Minimal performance impact. | Less accurate. Unless there's a consistent-hashing load balancer in front of Kong, it diverges when scaling the number of nodes.
+| `cluster` | Accurate<sup>1</sup>, no extra components to support. | Each request forces a read and a write on the data store. Therefore, relatively, the biggest performance impact. <br>Not supported in hybrid mode or {{site.konnect_short_name}} deployments. |
+| `redis`   | Accurate<sup>1</sup>, less performance impact than a `cluster` policy. | Needs a Redis installation. Bigger performance impact than a `local` policy. |
```

Comment:
I feel like we're missing a description of these, the pros and cons don't really tell me what these strategies actually do